components/salesforce_rest_api/README.md

```diff
@@ -4,6 +4,53 @@ You can install the Pipedream Salesforce app in the [Accounts](https://pipedream
 
 ### Webhooks
```

Comment:
I'd maybe make this H2 since you don't currently have one, and call it something more descriptive, like "Configuring Webhook Sources in Salesforce" or something